code/ui/manager/src/index.tsx

```diff
@@ -82,7 +82,8 @@ export function renderStorybookUI(domNode: HTMLElement, provider: Provider) {
     throw new Error('provider is not extended from the base Provider');
   }
 
-  ReactDOM.render(<Root key="root" provider={provider} />, domNode);
+  const root = createRoot(domNode!);
+  root.render(<Root key="root" provider={provider} />);
```

Comment:
Yeah I suppose this is correct, Let's see if it works 😅

Reply:
I hope it works! Thanks for your comment!